dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/Standard/Parameters.mo

```diff
@@ -0,0 +1,160 @@
+within Dynawo.Electrical.HVDC.Standard;
+
+/*
+* Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+package Parameters "Parameters of the HVDC Standard model"
+  extends Icons.RecordsPackage;
+
+  record Params_ACEmulation
+    import Dynawo.Types;
+    parameter Types.Time tFilter "Time constant of the angle measurement filter";
+    parameter Types.PerUnit KACEmulation "Inverse of the emulated AC reactance";
+  annotation(preferredView = "text");
+  end Params_ACEmulation;
+
+  record Params_RPFaultFunction
+    import Dynawo.Types;
+    parameter Types.PerUnit SlopeRPFault "Slope of the recovery of rpfault after a fault in p.u/s";
+  annotation(preferredView = "text");
+  end Params_RPFaultFunction;
+
+  record Params_DeltaP
+    import Dynawo.Types;
+    parameter Types.VoltageModulePu UdcMinPu "Minimum value of the DC voltage in p.u (base UNom)";
+    parameter Types.VoltageModulePu UdcMaxPu "Maximum value of the DC voltage in p.u (base UNom)";
+    parameter Types.PerUnit KpDeltaP "Proportional coefficient of the PI controller for the calculation of DeltaP";
+    parameter Types.PerUnit KiDeltaP "Integral coefficient of the PI controller for the calculation of DeltaP";
+  annotation(preferredView = "text");
+  end Params_DeltaP;
+
+  record Params_ActivePowerControl
+    import Dynawo.Types;
+    extends Params_DeltaP;
+    extends Params_RPFaultFunction;
+    parameter Types.PerUnit KpPControl "Proportional coefficient of the PI controller for the active power control";
+    parameter Types.PerUnit KiPControl "Integral coefficient of the PI controller for the active power control";
+    parameter Types.ActivePowerPu PMaxOPPu "Maximum operator value of the active power in p.u (base SNom)";
```

Comment:
With an import of Dynawo.Types at the beginning of the package it should make the job instead of duplicating it.

Reply:
Done